src/infra/service-infrastructure.md

```diff
@@ -62,23 +62,25 @@ wiki](https://github.com/rust-lang/triagebot/wiki) for more information.
 for Rust, allowing the Rust source tree to be navigated and searched with ease.
 It is generated by [rust-dxr](https://github.com/nrc/rust-dxr)
 
-## perf
+## perf / rust-timer
 
 [perf](https://github.com/rust-lang-nursery/rustc-perf) offers information
-about the performance of `rustc` over time.
+about the performance of `rustc` over time, and a bot for on-demand benchmarking.
 
-It is split into a data [collector] and a [web frontend]. The raw performance
+It is split into a data [collector] and a [web frontend + bot]. The raw performance
 data is available [here](https://github.com/rust-lang-nursery/rustc-timing)
 and can be browsed on the [perf website](https://perf.rust-lang.org).
 
-One-off performance runs can done by addressing the [rust-timer
-bot](https://github.com/rust-timer). On a PR, do a "try" build (`@bors try`),
-wait for it finish, then leave a comment with the merge commit hash like this:
+One-off performance runs can done by addressing the
+[rust-timer bot](https://github.com/rust-lang-nursery/rustc-perf/blob/master/site/src/github.rs)
+([bot user account](https://github.com/rust-timer)). On a PR, do a "try" build
+(`@bors try`), wait for it finish, then leave a comment with the merge commit
+hash like this:
```

Comment:
Could we update this with the new command? Specifically, you should now leave a single comment with something like `@bors try @rust-timer queue`. (Technically, the requirement is that the queue command finishes executing prior to the try command completing successfully).